src/templates/default/reviews/review_stages.html

```diff
@@ -0,0 +1,164 @@
+{% extends 'dashboard_base.html' %}
+
+{% load i18n static %}
+{% load proposals %}
+{% load compress crispy_forms_tags %}
+{% load tz %}
+{% get_current_timezone as TIME_ZONE %}
+
+{% block dashboard_tablist %}
+{% include '_includes/dashboard_tablist.html' with active='admin' %}
+{% endblock dashboard_tablist %}
+
+{% block main-content %}
+
+<h3 class="dashboard-header">
+	<span class="header__title">{% trans 'Review Stages' %}</span>
+</h3>
+
+<div class="dashboard-body">
+	<div class="row">
+		<!--Current Setting-->
+		<div class="panel panel-default">
+			<div class="panel-heading">{% trans 'Current Review Stage Setting' %}</div>
+			<ul class="list-group">
+				<li class="panel-body list-group-item">
+					{{ ".proposals.creatable"|configuration_switch }}
+					<div class="pull-right">
+						<p>{{ current_review_stages_setting.proposals_creatable }}</p>
+					</div>
+				</li>
+				<li class="panel-body list-group-item">
+					{{ ".proposals.editable"|configuration_switch }}
+					<div class="pull-right">
+						<p>{{ current_review_stages_setting.proposals_editable }}</p>
+					</div>
+				</li>
+				<li class="panel-body list-group-item">
+					{{ ".proposals.withdrawable"|configuration_switch }}
+					<div class="pull-right">
+						<p>{{ current_review_stages_setting.proposals_withdrawable }}</p>
+					</div>
+				</li>
+				<li class="panel-body list-group-item">
+					{{ ".proposals.reviews.visible.to.submitters"|configuration_switch }}
+					<div class="pull-right">
+						<p>{{ current_review_stages_setting.reviews_visible_to_submitters }}</p>
+					</div>
+				</li>
+				<li class="panel-body list-group-item">
+					{{ ".proposals.reviews.stage"|configuration_switch }}
+					<div class="pull-right">
+						<p>{{ current_review_stages_setting.reviews_stage }}</p>
+					</div>
+				</li>
+				<li class="panel-body list-group-item">
+					{{ ".proposals.proposals.disable.after"|configuration_switch }}
+					<div class="pull-right">
+						<p>{{ current_review_stages_setting.proposals_disable_after }}</p>
+					</div>
+			</ul>
+		</div>
```

Comment:
I think this whole part is kind of redundant because I can view current settings if they are loaded, which you have already added in line 156.

Reply:
Thank Tom. I Imagined these situations you mentioned and changed code done.
But proposals.disabled.after not loaded to settings, so I still let it display in current settings block.
![image](https://user-images.githubusercontent.com/20749936/104792132-583edf80-57d8-11eb-96dd-8a83d1686110.png)